certora/specs/UtilityFunctions.spec

```
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2025 Morpho Association
+
+function allocateOrDeallocate(bool isAllocate, env e, bytes data, uint256 assets, bytes4 selector, address sender) returns (bytes32[], int256) {
+    bytes32[] ids;
```

nitpick: could be like that
```suggestion
function allocateOrDeallocate(env e, bytes data, uint256 assets, bytes4 selector, address sender) returns (bytes32[], int256) {
    bool isAllocate;
    bytes32[] ids;
```